bokehjs/src/coffee/common/ui_events.coffee

```diff
@@ -16,38 +16,42 @@ class UIEvents extends Backbone.Model
     hit_area = @get('hit_area')
 
     unless options and options.testing
```

Comment:
"Unless" is synonymous with "if not".

Totally agree about stubbing. I forgot about that.